packages/uniforms-bootstrap5/src/BoolField.tsx

```diff
@@ -0,0 +1,61 @@
+import classnames from 'classnames';
+import React, { Ref } from 'react';
+import { connectField, HTMLFieldProps } from 'uniforms';
+
+import wrapField from './wrapField';
+
+export type BoolFieldProps = HTMLFieldProps<
+  boolean,
+  HTMLDivElement,
+  {
+    inline?: boolean;
+    inputClassName?: string;
+    inputRef?: Ref<HTMLInputElement>;
+    labelBefore?: string;
+  }
+>;
+
+function Bool({ onChange, ...props }: BoolFieldProps) {
+  const {
+    disabled,
+    error,
+    inline,
+    inputClassName,
+    inputRef,
+    label,
+    labelBefore,
+    name,
+    readOnly,
+    value,
+  } = props;
+  return wrapField(
+    { ...props, label: labelBefore, value: props.value },
+    <div
+      className={classnames(inputClassName, 'form-check', {
+        'text-danger': error,
+        'form-check-inline': inline,
+      })}
+    >
+      <label htmlFor={props.id} className="form-check-label">
+        <input
+          checked={value || false}
+          className="form-check-input"
+          disabled={disabled}
+          id={props.id}
+          name={name}
+          onChange={() => {
+            if (!readOnly) {
+              onChange(!value);
+            }
+          }}
+          ref={inputRef}
+          type="checkbox"
+        />
+        &nbsp;
+        {label}
+      </label>
```

Comment:
According to [docs](https://getbootstrap.com/docs/5.0/forms/checks-radios/), it should be:
```suggestion
      <input
        checked={value || false}
        className="form-check-input"
        disabled={disabled}
        id={props.id}
        name={name}
        onChange={() => {
          if (!readOnly) {
            onChange(!value);
          }
        }}
        ref={inputRef}
        type="checkbox"
      />
      <label htmlFor={props.id} className="form-check-label">
        {label}
      </label>
```

Reply:
@radekmie just wonder about `id` do we add it automatically or a user has to add this prop manually each time when he uses this `BoolField`?

Reply:
If it's provided, we use that one. Otherwise, there's a generated one in [`useField`](https://github.com/vazco/uniforms/blob/master/packages/uniforms/src/useField.tsx#L70).

Reply:
@radekmie do we need to use `&nbsp;.` we have a test for this
```
test('<BoolField> - renders a input with correct label (specified)', () => {
  const element = <BoolField name="x" label="BoolFieldLabel" />;
  const wrapper = mount(element, createContext({ x: { type: Boolean } }));

  expect(wrapper.find('label')).toHaveLength(1);
  expect(wrapper.find('label').text()).toBe(' BoolFieldLabel'); // Label is prefixed with a &nbsp;.
  expect(wrapper.find('label').prop('htmlFor')).toBe(
    wrapper.find('input').prop('id'),
  );
});
```

Reply:
Just check, whether it's needed UI-wise. If not, we can remove it and correct the test.